status-im/status-react
# feat(swap): add link to provider terms and conditions

fixes #21310 

### Summary

This PR adds a link to the swap provider terms and conditions on the Set Spending Cap and Swap Confirmation screens.

https://github.com/user-attachments/assets/fdc022cf-12e3-4e1e-b462-00acac241872

https://github.com/user-attachments/assets/b5f2f439-df0a-4077-ab88-9f312dd8072b

#### Platforms

- Android
- iOS

##### Functional

- wallet / transactions

### Steps to test

- Open Status
- Go to swap
- Select and enter valid amounts
- Verify Terms and Conditions is working properly

status: ready
